src/tutorial/tutorial.h

```diff
@@ -26,6 +26,7 @@ class Tutorial final : public QObject {
   Q_INVOKABLE void play(Addon* tutorial);
   Q_INVOKABLE void stop();
   Q_INVOKABLE void allowItem(const QString& objectName);
+  Q_INVOKABLE QString currentTutorialId();
```

Comment:
Instead of exposing the ID, what about if you expose the whole `m_currentTutorial` ? And it can be a property:
`Q_PROPERY(Addon* currentTutorial MEMBER m_currentTutorial NOTIFY playingChanged)`

Reply:
That makes sense, but why `Q_PROPERTY(Addon*...)` instead of `Q_PROPERTY(AddonTutorial*...)`